scripts/deploy

```diff
@@ -60,14 +61,9 @@ def bump_version(current_version: Version, bump_part: str) -> Version:
         for part in final_releases_parts
     ]
     if bump_part in DEFAULT_SPECIAL_RELEASE_LETTER:
-        special_version_number = getattr(current_version, bump_part)
-        # only pre-release version return tuples, others return number
-        if special_version_number and bump_part == 'pre':
-            special_version_number = special_version_number[1]
-        if not special_version_number:
-            parts.append(f'{DEFAULT_SPECIAL_RELEASE_LETTER[bump_part]}1')
-        else:
-            parts.append(f'{DEFAULT_SPECIAL_RELEASE_LETTER[bump_part]}{special_version_number + 1}')
```

Comment:
This part is to retrieve the current version number and bump by 1. If we remove this part, you can no longer bump version for post, pre (rc), and dev release.

Reply:
Instead, you should add an option flag, for instance:
```
self.add_argument(
            '-d', '--append-datetime', type=bool, action='store_true', help='append datetime at the end of version, only applicable for dev'
        )
```
Then something like
```
special_version_number = getattr(current_version, bump_part)
        # only pre-release version return tuples, others return number
        if special_version_number and bump_part == 'pre':
            special_version_number = special_version_number[1]
       if use_datetime and bump_part == 'dev':
            parts.append(f'dev{datetime.utcnow()})
       elif not special_version_number:
            parts.append(f'{DEFAULT_SPECIAL_RELEASE_LETTER[bump_part]}1')
        else:
            parts.append(f'{DEFAULT_SPECIAL_RELEASE_LETTER[bump_part]}{special_version_number + 1}')
```

Reply:
A better way is handling the flag at parser, then use a seprate function from bump version where you check, for instance, that you can only append date with dev version.